backend/corpora/dataset_processing/make_seurat.R

```diff
@@ -1,5 +1,8 @@
 library(sceasy)
 
+require(devtools)
+install_version("SeuratObject", version = "4.0.2", repos = "http://cran.rstudio.com/")
```

Comment:
Does it actually force re-installation? 

I think a better place for this may be  when the docker image is created.

https://github.com/chanzuckerberg/single-cell-data-portal/blob/c12018441a6037bc3b4c50d1381fd84e4065b0a1/Dockerfile.processing_base#L27

Reply:
Yes, this forces reinstallation. I don't think we should put that in `processing_base`, since this is meant to be a temporary solution. Also, `processing_base` is pushed on a schedule and this fix needs to go in ASAP. Happy to move this around later.

Reply:
okay! SGTM then